### PR TITLE
Issue 11

### PR DESCRIPTION
**Pull request by derekjamerson**
#11

### CODE REVIEW

File: deck.py [hunk 1]
```diff
@@ -5,7 +5,21 @@
 class Deck:
     suits = ["Clubs", "Hearts", "Spades", "Diamonds"]
 
-    values = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"]
+    values = [
```
Comment (jlward) [inline deck.py:8]: You are going to get a merge conflict here.

File: main.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from game import Game
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```
Comment (jlward) [inline main.py:4]: `black` started a holy war about which was better `'` or `"`. Black chooses `"`. It's really unfortunate. Cause I hate it. Lol. I always use [skip-string-normilzation](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings). Long story short, be consistent, and it doesn't really matter.

Comment (derekjamerson) [reply main.py:4]: L O L

File: deck.py (hunk 1 above)
Comment (derekjamerson) [reply deck.py:8]: I can manually resolve that though, right?

Comment (jlward) [reply deck.py:8]: Yes. When you merge this branch up with `master`, you'll get conflicts. My money says `PyCharm` has a way to help you deal with it. If it doesn't, you can handle it in the terminal. If you have problems figuring out how to interpret the merge conflicts, let me know. They can be confusing.

Comment (derekjamerson) [reply deck.py:8]: I thought Github would flag the conflict and help me resolve it

Comment (jlward) [reply deck.py:8]: GH did flag it. IF you click on the conversation tab and scroll to the bottom it'll show up. GH might have a way to help, I've never used it before. We can jump on a video call if you need help.

Comment (derekjamerson) [reply deck.py:8]: Nah, I think I got it. I have used the GH editor before.

**Pull request merged**